### PR TITLE
Website fixups for admin partitions

### PR DESCRIPTION
**Pull request by markan**
Quick pass through docs

Signed-off-by: Mark Anderson <manderson@hashicorp.com>

### CODE REVIEW

**Review by rboyer — APPROVED** [review]

File: website/content/commands/services/deregister.mdx [hunk 1]
```diff
@@ -36,6 +36,8 @@ This flexibility makes it easy to pair the command with the
 
 @include 'http_api_namespace_options.mdx'
 
+@include 'http_api_partitions_options.mdx'
```
Comment (freddygv) [inline website/content/commands/services/deregister.mdx:39]: There's a mix of spellings here, some of these refs say `http_api_partitions_options` while others say `http_api_partition_options`. The filename with the partials is singular.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/529458.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9806b7c268afa4d749a670911202c6c6648a39d8 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 9806b7c268afa4d749a670911202c6c6648a39d8 onto `release/1.11.x` succeeded!

**Pull request merged**